schemas/viewdefinition/0.0.1-preview/ViewDefinition.json

```diff
@@ -212,18 +212,24 @@
                         },
                         "resourceType": {
                             "type": "string",
-                            "description": "The Custom Resource type."
+                            "description": "The custom resource type."
                         },
                         "createUIDefinition": {
                             "type": "object",
-                            "description": "Create UI Definition schema for create custom resource command."
+                            "description": "The Create UI Definition schema for create custom resource command.",
+                            "$ref": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json"
```

Comment:
>https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json [](start = 37, length = 89)

this is a reference to the whole file. is that expected?

Reply:
yes, we are referencing external schema here, "createUIDefinition" content supports the same properties as createUiDefinition.json file